packages/mdc-slider/_mixins.scss

```diff
@@ -435,8 +435,16 @@
   $feat-color: mdc-feature-create-target($query, color);
 
   .mdc-slider__track-container {
-    @include mdc-feature-targets($feat-color) {
-      @include mdc-theme-prop(background-color, rgba(mdc-theme-prop-value($color), $opacity));
+    &::after {
```

Comment:
This PR does not seem to change the color of slider to secondary color?

Reply:
Add the following to the test/screenshot/spec/mdc-slider/clases/baseline_continuous.html:

Inside of the head tag:
```
<style>
  .test-secondary-color {
    --mdc-theme-secondary: orange;
  }
</style>
```

Inside of the body, add a second slider with the test-secondary-color class:
```
<div class="mdc-slider test-slider test-secondary-color" tabindex="0" role="slider"
     aria-valuemin="0" aria-valuemax="100" aria-valuenow="0"
     aria-label="Select Value">
  <div class="mdc-slider__track-container">
    <div class="mdc-slider__track"></div>
  </div>
  <div class="mdc-slider__thumb-container">
    <svg class="mdc-slider__thumb" width="21" height="21">
      <circle cx="10.5" cy="10.5" r="7.875"></circle>
    </svg>
    <div class="mdc-slider__focus-ring"></div>
  </div>
</div>
```

Then run `npm start` and navigate to that page.  You will see the added slider use the secondary color.